README.md

```
@@ -136,6 +136,22 @@ powershell.exe -STA -File .\GUI\Start-SysAdminSuiteGui.ps1
 
 > **Why `-STA`?** WinForms requires Single Threaded Apartment mode. Without it the window won't render.
 
+### Launch the Web Dashboard (no PowerShell)
+
+For endpoints where `powershell.exe` is blocked or constrained, the dashboard ships
+as a standalone .NET 8 tray host. Double-click:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)